dockstore-integration-testing/src/test/java/io/dockstore/webservice/helpers/GitHubAppHelper.java

```diff
@@ -56,6 +56,16 @@ public static void handleGitHubRelease(WorkflowsApi workflowsApi, String reposit
     public static void handleGitHubInstallation(WorkflowsApi workflowsApi, List<String> repositories, String gitHubUsername) {
         InstallationRepositoriesPayload payload = new InstallationRepositoriesPayload()
                 .repositoriesAdded(repositories.stream().map(repo -> new WebhookRepository().fullName(repo)).toList());
+        payload.setAction(io.dockstore.webservice.core.webhook.InstallationRepositoriesPayload.Action.ADDED.toString());
```

Comment:
Could import `io.dockstore.webservice.core.webhook.InstallationRepositoriesPayload` to make this a bit more concise, not worth changing unless there are other changes